plugins-scripts/t/check_file_age.t

```diff
@@ -48,22 +48,44 @@ cmp_ok( $result->return_code, '==', 1, "Warning for file over 1 second old" );
 like  ( $result->output, $warningOutput, "Output for warning correct" );
 
 $result = NPTest->testCmd(
+	"./check_file_age -f $temp_file -w 0:1"
```

Comment:
The tests will fail if Monitoring::Plugins::Range is not installed. Could you add a `apt-get install libmonitoring-plugin-perl` to the .travis.yml file?